pkgs/shells/zsh/lambda-mod-zsh-theme/default.nix

```diff
@@ -10,9 +10,18 @@ stdenv.mkDerivation {
     rev = "61c373c8aa5556d51522290b82ad44e7166bced1";
   };
 
+  buildInputs = [ zsh ];
+
   installPhase = ''
-    mkdir -p $out/share/themes
-    cp lambda-mod.zsh-theme $out/share/themes
+    runHook preInstall
+
+    chmod +x lambda-mod.zsh-theme # only executable scripts are found by `patchShebangs`
+    patchShebangs .
+    chmod -x lambda-mod.zsh-theme
```

Comment:
I think it's okay to let it stay executable, it is executable after all, especially since the shebang was patched.